auto_nag/scripts/uplift_beta.py

```diff
@@ -0,0 +1,156 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import dateutil.parser
+from libmozdata import utils as lmdutils
+from libmozdata.bugzilla import Bugzilla
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+class UpliftBeta(BzCleaner):
+    def __init__(self):
+        super(UpliftBeta, self).__init__()
+        if not self.init_versions():
+            return
+
+        self.beta = self.versions["beta"]
+        self.status_central = utils.get_flag(
+            self.versions["central"], "status", "central"
+        )
+        self.status_beta = utils.get_flag(self.beta, "status", "beta")
+
+    def description(self):
+        return "Bugs fixed in nightly but still affect other supported channels"
+
+    def has_assignee(self):
+        return True
+
+    def columns(self):
+        return ["id", "summary", "assignee"]
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+
+        assignee = bug.get("assigned_to", "")
+        if utils.is_no_assignee(assignee):
+            assignee = ""
+            nickname = ""
+        else:
+            nickname = bug["assigned_to_detail"]["nick"]
+
+        data[bugid] = {
+            "id": bugid,
+            "mail": assignee,
+            "nickname": nickname,
+            "summary": self.get_summary(bug),
+            "regressions": bug["regressions"],
+        }
+        return bug
+
+    def filter_bugs(self, bugs):
+        # Get the bugs where status_central has been set few days ago
```

Comment:
Can't we rely on status moving to FIXED, and so bake this directly in the Bugzilla query?